### PR TITLE
gx_save + gx_restore to save and restore .RHistory and .RData

### PR DESCRIPTION
**Pull request by scholtalbers**
Make the convience gx_save more convenient by also storing the .RHistory file and allow .RData and .RHistory to be restored by gx_restore.

### CODE REVIEW

File: GalaxyConnector/R/GalaxyConnector.R [hunk 1]
```diff
@@ -26,10 +26,30 @@ gx_get <- function(file_id){
...
+    savehistory(hist)
+    gx_put(workspace)
+    gx_put(hist)
+}
+
+
+#' gx_restore
+#'
+#' Restore the notebook from a .RData and .RHistory object from the current Galaxy history.
+#' Convenience function which wraps save.image and gx_put
```
Comment (bgruening) [inline GalaxyConnector/R/GalaxyConnector.R]: save.image is not wrapped. I suppose this is a c&p error.

**Comment by bgruening** [comment]
LGTM. Thanks  @scholtalbers.

**Comment by erasche** [comment]
Thanks @scholtalbers I'd been meaning to get around to this for a long time now.

File: GalaxyConnector/R/GalaxyConnector.R (hunk 1 above)
Comment (erasche) [inline GalaxyConnector/R/GalaxyConnector.R]: yep, delete this line and I'll merge.

**Comment by scholtalbers** [comment]
Thanks to @erasche comments I understood what you meant @bgruening - thought somehow i forgot to wrap something in the function :)

**Comment by bgruening** [comment]
:+1:

**Pull request merged**